Sources/Extensions/UIKit/UIViewExtensions.swift

```diff
@@ -464,6 +464,36 @@ public extension UIView {
 		}
 	}
 
+    /// SwifterSwift: Anchors all sides to superview's safe area.
+    ///
+    /// 1. On iOS 11 and latter anchors view's sides to superview's safe area
+    /// 2. On iOS 9-11 anchors leading and trailing to superview,
+    /// top and bottom to viewController's bottomLayoutGuide and topLayoutGuide.
+    ///
+    /// - Parameter viewController: view controller, of which bottomLahoutGuide
+    ///  and topLayoutGuide are used on iOS 9-11
```

Comment:
By naming convention I'm not sure if this is expressive enough, maybe something like `constraintEquallyToSuperview()`, because it express more the idea that the it rely on contraints. 
Just a suggestion, let me know you opinion on that :))

Reply:
Don't think we need to express `equally` specifically. Something like `constrainToSuperview`, or `fillSuperView` should be a good name. I think the best names are descriptive while not being overly explicit

Reply:
Ok so we already have `fillToSuperview` in SwifterSwift. I think `fillToSuperviewSafeArea` is a good name for this

Reply:
@LucianoPAlmeida I actually agree with you: `constraintViewToSuperviewSafeArea` is a better name than `fillToSuperviewSafeArea` since it uses the same amount of words, but gives us more information about the result of function's execution (I mean that it uses autolayout and not frames or autoresizing masks). 
I think it's worth considering changing this function's name together with already existing `fillToSuperview`.

Reply:
Hey @viktart, sorry for the delay on responding 😭 
But for sure, It's worth considering but since we found that we already have `fillToSuperview`, renaming it will be a breaking change to take into account too :))